rdf_entity.module

```diff
@@ -195,18 +198,24 @@ function rdf_entity_form_alter(&$form, FormStateInterface $form_state) {
     '#tree' => TRUE,
   ];
 
-  if (!$mapping = RdfEntityMapping::loadByName($entity_type_id, $bundle_entity->id())) {
-    $mapping = RdfEntityMapping::create([
-      'entity_type_id' => $entity_type_id,
-      'bundle' => $bundle_entity->id(),
-    ]);
+  // When creating a new bundle we can't yet use it for creating a mapping. In
+  // this case we defer to the custom submit handler..
```

Comment:
This is not an error for code sniffer nut still: there are 2 dots at the end of the phrase.